cmd/adapter-rest/startcmd/start_test.go

```diff
@@ -567,7 +598,26 @@ func TestAdapterModes(t *testing.T) { // nolint:paralleltest // shared environme
 		require.Contains(t, err.Error(), "failed to initialize wallet bridge")
 	})
 }
-
+func TestReadOutputDescriptorFile(t *testing.T) { // nolint:paralleltest
+	t.Run("read output descriptor file success", // nolint:paralleltest
+		func(t *testing.T) {
+			cmOutputdesc, err := readCMOutputDescriptorFile("./testdata/outputdescriptors.json")
+			require.NoError(t, err)
+			require.NotNil(t, cmOutputdesc)
+			for k := range cmOutputdesc {
```

Comment:
u can do `for k, v := range cmOutputdesc`

Reply:
also test size of result map, to make sure you are losing any data.

Reply:
done